modules/common/development/scripts/perf_test_icicle_kit.nix

```diff
@@ -0,0 +1,20 @@
+# Copyright 2022-2024 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+{pkgs, ...}:
+with pkgs;
```

Comment:
Don't use `with pkgs;`

Reply:
This also applies to all other files, I just don't want to add 4 comments with same text